### ISSUE TITLE
Problem querying floating point number fields with where. 

### ISSUE THREAD

**Issue by SeriousKen** [dwgebler/doclite#14]
If you have a floating point number in a document you can't query it using `where`

``` php
$doclite = new FileDatabase('database.db');
$products = $doclite->collection('products');

$product = $products->get();
$product->name = 'Product 1';
$product->price = 10.50;
$product->save();

// Outputs 1
echo iterator_count($products->findAllBy(['price' => 10.50])) . "\n";

// Outputs 0
echo $products->where('price', '=', 10.50)->count() . "\n";
```

**Comment by dwgebler** [comment]
Thanks for the bug report @SeriousKen I'm looking in to it.

**Comment by dwgebler** [comment]
@SeriousKen should be fixed in release 1.1.4. Please update to the new version and let me know if you encounter any further issues, thanks.
